heron/common/src/cpp/metrics/metricsmgr-client.cpp

```diff
@@ -63,6 +63,12 @@ void MetricsMgrClient::HandleClose(NetworkErrorCode) {
 
 void MetricsMgrClient::ReConnect() { Start(); }
 
+void MetricsMgrClient::SetPublisherPort(const sp_int32 _port) {
+  if (0 == port_ && 0 != _port) {
```

Comment:
The `port_` may be 0 or not be 0.
In the unit test, it is initialized as 0 through constructor parameter.
If it is not initialized as 0, it will not be set to other value during its lifetime, which is the same behavior as the before.